src/haproxy/haproxy.cfg

```diff
@@ -42,6 +46,11 @@ backend redirect-to-https-in
   balance roundrobin
   server localhost 127.0.0.1:444 send-proxy-v2
 
+backend redirect-to-tunnel-in
+  mode tcp
+  balance roundrobin
+  server localhost 127.0.0.1:3129
```

Comment:
hmm 3129? Just checking: in balenaCloud we decided to use port 443 with `tunnel.balena-cloud.com` and `tunnel.balena-staging.com`. At one point we discussed enabling TLS on port 3129, but then we changed it to port 443.

Reply:
this isn't what you think it is @pdcastro it's just how we have to wire HAProxy. Look in the `https-in` frontend and you'll see that the SNI is checked and sent to this backend as you cant do a HTTP frontent to a TCP backend.

Reply:
I see, and haproxy itself creates a socket listening on port 3129? I see a `bind *:3129` line later in the file. I didn't know that haproxy could do that. 👍